src/ResourceManager/EventGrid/Commands.EventGrid/EventSubscription/UpdateAzureEventGridSubscription.cs

```diff
@@ -206,6 +206,87 @@ public class UpdateAzureEventGridSubscription : AzureEventGridCmdletBase
         [ValidateNotNullOrEmpty]
         public string[] Label { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
```

Comment:
Same comment in this file about remove position for non-mandatory parameters that have just been added.

Reply:
done